src/dmd/backend/dwarfdbginf.d

```diff
@@ -483,105 +494,112 @@ struct Section
-static Section debug_aranges  = { 0,0,0, ".debug_aranges", SHT_PROGBITS };
-static Section debug_ranges   = { 0,0,0, ".debug_ranges", SHT_PROGBITS };
-static Section debug_loc      = { 0,0,0, ".debug_loc", SHT_PROGBITS };
-static Section debug_abbrev   = { 0,0,0, ".debug_abbrev", SHT_PROGBITS };
-static Section debug_info     = { 0,0,0, ".debug_info", SHT_PROGBITS };
-static Section debug_str      = { 0,0,0, ".debug_str", SHT_PROGBITS };
-static Section debug_line     = { 0,0,0, ".debug_line", SHT_PROGBITS };
-#endif
-
-#if MACHOBJ
-const char* debug_frame_name = "__debug_frame";
-#elif ELFOBJ
-const char* debug_frame_name = ".debug_frame";
-#endif
+    Section debug_line     = { 0,0,null, "__debug_line", S_REGULAR };
+}
+else static if (ELFOBJ)
+{
+    Section debug_pubnames = { 0,0,null, ".debug_pubnames", SHT_PROGBITS };
+    Section debug_aranges  = { 0,0,null, ".debug_aranges", SHT_PROGBITS };
+    Section debug_ranges   = { 0,0,null, ".debug_ranges", SHT_PROGBITS };
+    Section debug_loc      = { 0,0,null, ".debug_loc", SHT_PROGBITS };
+    Section debug_abbrev   = { 0,0,null, ".debug_abbrev", SHT_PROGBITS };
+    Section debug_info     = { 0,0,null, ".debug_info", SHT_PROGBITS };
+    Section debug_str      = { 0,0,null, ".debug_str", SHT_PROGBITS };
+    Section debug_line     = { 0,0,null, ".debug_line", SHT_PROGBITS };
+}
+
+static if (MACHOBJ)
+    const char* debug_frame_name = "__debug_frame";
+else static if (ELFOBJ)
+    const char* debug_frame_name = ".debug_frame";
 
 
 /* DWARF 7.5.3: "Each declaration begins with an unsigned LEB128 number
  * representing the abbreviation code itself."
  */
-static uint abbrevcode = 1;
-static AApair *abbrev_table;
-static int hasModname;    // 1 if has DW_TAG_module
+uint abbrevcode = 1;
+AApair *abbrev_table;
+int hasModname;    // 1 if has DW_TAG_module
 
 // .debug_info
-static AAchars *infoFileName_table;
+AAchars *infoFileName_table;
 
-static AApair *type_table;
-static AApair *functype_table;  // not sure why this cannot be combined with type_table
-static Outbuffer *functypebuf;
+AApair *type_table;
+AApair *functype_table;  // not sure why this cannot be combined with type_table
+Outbuffer *functypebuf;
 
-#pragma pack(1)
 struct DebugInfoHeader
-{   uint total_length;
-    ushort version;
+{
+  align (1):
+    uint total_length;
+    ushort _version;
     uint abbrev_offset;
     ubyte address_size;
-};
-#pragma pack()
+}
 
-static DebugInfoHeader debuginfo_init =
+DebugInfoHeader debuginfo_init =
 {       0,      // total_length
-        3,      // version
+        3,      // _version
         0,      // abbrev_offset
         4       // address_size
 };
 
-static DebugInfoHeader debuginfo;
+DebugInfoHeader debuginfo;
 
 // .debug_line
-static size_t linebuf_filetab_end;
+size_t linebuf_filetab_end;
 
-#pragma pack(1)
 struct DebugLineHeader
-{   uint total_length;
-    ushort version;
+{
+  align (1):
+    uint total_length;
+    ushort _version;
```

Comment:
The convention is usually to suffix the name with an underscore instead of prefix it if it's has the same name as a D keyword [1].

[1] https://dlang.org/dstyle.html#naming_keywords